sklearn/lda.py

```diff
@@ -1,6 +1,20 @@
 import warnings
+from .discriminant_analysis import LinearDiscriminantAnalysis as _LDA
```

Comment:
But the whole lda module is deprecated and will be removed (and that is included in the warning). Also, this was a change in 0.17